docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1651 @@
+{
```

Comment:
Could you please check it out again? 

Because, I think we were correct and you did not make a mistake. So, for instance, let's say `cand_idx_chunks`  is `[10,11,15]` . That means I need to find the distance of three subsequences (with **START** index `10`, `11`, and `15`) to Q.  The chunks will be `[10,12]` and  `[15,16]`.  So, for the first chunk, we should create a time series that contains subsequences starting at `10` and `11`.   So, the time series will be `T[start: (stop - m + 1)]=T[10: (12 + m - 1)]`.

Reply:
<p>Ahhhhh, I see my mistake! We are going from "subsequence index" space to "time series index" space. Thank you for explaining that. I am pretty sure that I will make this mistake again in the future. :)</p>

Reply:
I have been thinking about it as I do not want the reader to get confused later. 

So, I can do:
* use a new (temporary) variable `end_idx  = stop + m - 1` 
* Or, I can add a comment 

I think the the second one is better. If you have any suggestion, please let me know.

Reply:
<p>I think a simple comment like:</p><pre class="ql-syntax" spellcheck="false"># Note: <code>stop + m - 1</code> is correct for converting <code>stop</code> from "subsequence index" space to "time series index" space. Do not replace with the incorrect <code>stop - m + 1</code> in this case!
</pre>